code/modules/admin/verbs/pray.dm

```diff
@@ -29,6 +29,10 @@
 		usr.whisper(orig_message)
 	to_chat(usr, "Your prayers have been received by the gods.")
 
+	if(lowertext(ticker.Bible_deity_name) == "buttbot" || lowertext(ticker.Bible_deity_name) == "butt bot")
```

Comment:
ticker.Bible_deity_name is bad because it's from the old pre labor system where you could only have one chappie. It gets overwritten when any new chaplain joins

You could search all active religions to see if any God is buttbot and then reply automatically, or you could even use DecidePrayerGod and only buttify if they are faithful to buttbot or have no religion (that way it doesn't butt up people praying to Narnar or their specific non-buttbot God)

Reply:
You sure about this one? The pray verb itself ticker.Bible_deity_name, the admin reply button uses ticker.Bible_deity_name, and I can't find DecidePrayerGod or similar anywhere.

Reply:
Religions.dm line 1

Reply:
Please be more specific, I still can't find the proc you were talking about or anywhere that this sets global variables.

Reply:
Oh I was on a 2 month old branch